ProductsAnalysisWeb/Controllers/MobileController.cs

```diff
@@ -19,25 +19,25 @@ public IActionResult Home()
         }
         public IActionResult Shop()
         {
-            List<Mobile> n = new List<Mobile>();
+            List<Mobile> mobilesList = new List<Mobile>();
             if (Start.CheckXmlExists())
             {
-                n = Start.ExtractProductsFromeXML();
+                mobilesList = Start.FromeXML();
```

Comment:
I was unclear 🥴 maybe ... Original suggestion for the method was 'ExtractProducts' or 'GetProducts'